scripts/ansible

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Note: don't forget to login to registry.gitlab.com: `docker login registry.gitlab.com`
```

Comment:
Why do you need to login into `gitlab`?

Reply:
I just used this template from oai blueprint so i didn't change much. I will check this and update